CanHazFunny/CanHazFunny/IJokeDisplay.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny
+{
+    public interface IJokeDisplay
+    {
+        public void DisplayJoke(string joke);

```

Comment:
Avoid specifying access modifiers on interfaces
```suggestion
        void DisplayJoke(string joke);

```